src/seismometer/data/performance.py

```diff
@@ -360,6 +363,9 @@ def calculate_bin_stats(
                 1 - fpr,  # Specificity
                 ppv,
                 npv,
+                f_1,
```

Comment:
```suggestion
                f1,
```
It matches what you have above.